Strata/Languages/Laurel/LaurelToBoogieTranslator.lean

```diff
@@ -0,0 +1,89 @@
+/-
+  Copyright Strata Contributors
+
+  SPDX-License-Identifier: Apache-2.0 OR MIT
+-/
+
+import Strata.Languages.Boogie.Program
+import Strata.Languages.Boogie.Verifier
+import Strata.Languages.Boogie.Statement
+import Strata.Languages.Boogie.Procedure
+import Strata.Languages.Boogie.Options
+import Strata.Languages.Laurel.Laurel
+
+namespace Laurel
+
+open Boogie (VCResult VCResults)
+open Strata
+
+/-
+Translate Laurel StmtExpr to Boogie Expression
+-/
+partial def translateExpr (expr : StmtExpr) : Boogie.Expression.Expr :=
```

Comment:
I imagine this will ultimately return two things: a list of statements to execute first, and an expression that contains the value after executing those statements. But that's not needed yet.